python/nav/adapters.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2018 UNINETT AS
+#
+# This file is part of Network Administration Visualized (NAV).
+#
+# NAV is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License version 2 as published by the Free
+# Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.  You should have received a copy of the GNU General Public
+# License along with NAV. If not, see <http://www.gnu.org/licenses/>.
+#
+
+from __future__ import print_function, absolute_import
+
+try:
+    from django.contrib.postgres.fields import HStoreField
+    DictionaryField = HStoreField
+    print('Using new style hstore: django.contrib.hstore')
```

Comment:
No printing in production code, please :)

Reply:
Fixed in https://github.com/UNINETT/nav/pull/1684/commits/eb9e910710ed4363b50a1467ded32fd6d8e96926 .